### PR TITLE
cargo  crate fix

### LINKED ISSUE #43487: Minor Issues brought by #43454
@Tlaltecuhtli 

Some issues from testing:

1. SecTech supply crate spawns a SecTech vendor. 
2. Spawning `/obj/item/vending_refill/security` creates an invisible Generic restocking unit. 
3. New crates are unsorted and should go to their alphabetic spots.

_Originally posted by @Angustmeta in https://github.com/tgstation/tgstation/pull/43454#issuecomment-479954835_

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
fix: cargo sectech crate
fix: sectech refill has right icon
/:cl:

Fixes #43487

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -419,7 +419,7 @@
 	name = "SecTech Supply Crate"
 	desc = "Officer Paul bought all the donuts? Then refill the security vendor with ths crate."
 	cost = 1500
-	contains = list(/obj/machinery/vending/security)
+	contains = list(/obj/item/vending_refill/security)
```
Comment (SpaceManiac) [inline code/modules/cargo/packs.dm]: nice

**Review by SpaceManiac — DISMISSED** [review]

**Comment by Angustmeta** [comment]
Sort crates by alphabet please, or they may spiral out of control later. Looks nicer in-game that way also.

**Comment by Tlaltecuhtli** [comment]
i ve organized aphabetically (80% of those crates werent even mine) and put the vendor refills at the end of each category

**Comment by ATH1909** [comment]
> i ve organized aphabetically (80% of those crates werent even mine) and put the vendor refills at the end of each category

Not all heroes wear capes.

**Comment by AnturK** [comment]
Things shouldn't depend on compile order, if there's some missing sorting add it where it's needed.

**Pull request merged**